internal/am/job_tracker_test.go

```diff
@@ -161,3 +173,91 @@ func TestJobTracker(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertJobToPreservationTask(t *testing.T) {
+	t.Parallel()
+	var og time.Time
+	jobs := []amclient.Job{
+		{
+			ID:           "f60018ac-da79-4769-9509-c6c41d5efe7e",
+			LinkID:       "70669a5b-01e4-4ea0-ac70-10292f87da05",
+			Microservice: "Verify SIP compliance",
+			Name:         "Move to processing directory",
+			Status:       amclient.JobStatusComplete,
+			Tasks: []amclient.Task{
+				{
+					ID:          "c134198c-9485-4f68-8d94-4da1e03b5e1b",
+					ExitCode:    0,
+					CreatedAt:   amclient.TaskDateTime{Time: time.Date(2024, time.January, 18, 1, 27, 49, 0, time.UTC)},
+					StartedAt:   amclient.TaskDateTime{Time: time.Date(2024, time.January, 18, 1, 27, 49, 0, time.UTC)},
+					CompletedAt: amclient.TaskDateTime{Time: time.Date(2025, time.January, 18, 1, 27, 49, 0, time.UTC)},
+					Duration:    amclient.TaskDuration(time.Second / 2),
+				},
+			},
+		},
+		{
+			ID:           "c2128d39-2ace-47c5-8cac-39ded8d9c9ef",
+			LinkID:       "208d441b-6938-44f9-b54a-bd73f05bc764",
+			Microservice: "Verify SIP compliance",
+			Name:         "Verify SIP compliance",
+			Status:       amclient.JobStatusComplete,
+			Tasks: []amclient.Task{
+				{
+					ID:          "6f5beca3-71ad-446c-8f19-3bc4dea16c9b",
+					ExitCode:    0,
+					CreatedAt:   amclient.TaskDateTime{Time: time.Date(2024, time.January, 18, 1, 27, 49, 0, time.UTC)},
+					StartedAt:   amclient.TaskDateTime{Time: time.Date(2024, time.January, 18, 1, 27, 49, 0, time.UTC)},
+					CompletedAt: amclient.TaskDateTime{Time: og},
+					Duration:    amclient.TaskDuration(time.Second / 2),
+				},
+			},
+		},
+	}
+	type test struct {
+		name string
+		job  amclient.Job
+		want []sql.NullTime
```

Comment:
If you're testing ConvertJobToPreservationTask then you should be looking at its return value (package_.PreservationTask) and not just part of it.